deployment/ansible/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+inventory = inventory/
```

Comment:
Is this not already the default, or at least a place Ansible knows to look at on its own?